src/metabase/models/params.clj

```diff
@@ -77,7 +78,8 @@
 
 (defn- field-ids->param-field-values-ignoring-current-user
   [param-field-ids]
-  (t2/select-fn->fn :field_id identity ['FieldValues :values :human_readable_values :field_id]
+  (t2/select-fn->fn :field_id (comp identity t2.realize/realize)
```

Comment:
need to realize the row because apparently `select-fn->fn` does not realize it for the xform.

filed an issue here :https://github.com/camsaul/toucan2/issues/144